Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -344,26 +344,40 @@ def condition_(self, fragment, stepList, key, hasHarvest):
 
 class UpgradeWorkflow_ProdLike(UpgradeWorkflow):
     def setup_(self, step, stepName, stepDict, k, properties):
-        if 'Reco' in step:
+        if 'Digi' in step and 'Trigger' not in step:
+            stepDict[stepName][k] = merge([{'-s': 'DIGI,L1,DIGI2RAW,HLT:@relval2021', '--datatier':'GEN-SIM-DIGI-RAW', '--eventcontent':'RAWSIM'}, stepDict[step][k]])
+        elif 'Reco' in step:
             stepDict[stepName][k] = merge([{'-s': 'RAW2DIGI,L1Reco,RECO,RECOSIM', '--datatier':'AODSIM', '--eventcontent':'AODSIM'}, stepDict[step][k]])
         elif 'MiniAOD' in step:
             # the separate miniAOD step is used here
             stepDict[stepName][k] = deepcopy(stepDict[step][k])
-        if 'HARVEST' in step:
+        if 'ALCA' in step or 'HARVEST' in step:
             # remove step
             stepDict[stepName][k] = None
+	if 'Nano' in step:
```

Comment:
please fix the indentation - currently uses a tab, but spaces should be used in python for consistency